packages/vertexai/src/requests/stream-reader.ts

```diff
@@ -93,9 +94,10 @@ export function getResponseStream<T>(
           if (done) {
             if (currentText.trim()) {
               controller.error(
-                ERROR_FACTORY.create(VertexError.PARSE_FAILED, {
-                  message: 'Failed to parse stream'
-                })
+                new VertexAIError(
+                  VertexAIErrorCode.PARSE_FAILED,
```

Comment:
This is actually one case that the previous template doesn't add anything. It would be "Parsing failed: Failed to parse stream" which is kind of redundant, so don't bother to add it here. Same with the next error on 114.

Reply:
The message field is required right now, so we can't remove the message. Should we make `message` optional?

Reply:
Oh sorry, I meant to say don't change anything here, it's fine, I wanted to explicitly say these two can be left alone, because I told you to change it everywhere else.